src/plugins/simulator/CMakeLists.txt

```diff
@@ -37,10 +37,13 @@ endif()
 
 if(SWIFT_BUILD_MSFS_PLUGIN)
     add_subdirectory(msfs)
-    add_subdirectory(msfsconfig)
+ endif()
+
+if(SWIFT_BUILD_MSFS2024_PLUGIN)
+    add_subdirectory(msfs2024)
 endif()
 
 if(SWIFT_BUILD_FSX_PLUGIN)
     add_subdirectory(fsx)
     add_subdirectory(fsxconfig)
-endif()
+endif()
```

Comment:
Here a newline (\n) at the end of the file is missing

Reply:
added. hopefully correct